### PR TITLE
Task 8

### PR DESCRIPTION
**Pull request by NattalZ30**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -105,6 +105,39 @@ describe("CORE:",() => {
...
+                .patch("/api/articles/1")
+                .send({
+                    inc_votes : 1 
+                })
+                .expect(200)
+                .then((response) => {
+                    const {
+                        body: { update },
+                    } = response;
+                    expect(update).toHaveLength(1)
```
Comment (nc-christian-dutton) [inline __tests__/app.test.js:120]: Shouldn't need to check the length as you shouldn't need to send an array back, return rows[0] in your model to just get the object back

File: __tests__/app.test.js [hunk 2]
```diff
@@ -105,6 +105,39 @@ describe("CORE:",() => {
...
+                    expect(update[0].votes).toBe(101);
+                })
+            }),
+            it("400: returns Error message", () => {
```
Comment (nc-christian-dutton) [inline __tests__/app.test.js:124]: Be more descriptive!

File: __tests__/app.test.js [hunk 3]
```diff
@@ -105,6 +105,39 @@ describe("CORE:",() => {
...
+            })
+            it("404: returns Error message", () => {
+                return request(app)
+                .patch("/api/articles/100000000")
+                .expect(404)
+                .then((response) => {
+                    expect(response.body.msg).toBe("404: NOT FOUND")
+                })
+            })
+        })
```
Comment (nc-christian-dutton) [inline __tests__/app.test.js:140]: There's more tests you can add here, what if inc_votes is not a number or you send an empty object?

File: models/app-models.js [hunk 4]
```diff
@@ -73,4 +73,11 @@ exports.insertCommentByArticle = async (article_id, comment) => {
                 const newRows = convertDateToTimestamp(rows[0])
                 return [newRows]
             })
+}
+
+exports.updateArticleById = async (article_id, update) =>{
+    const queryVal = [update.inc_votes, article_id]
+    await checkExists("articles","article_id",article_id)
```
Comment (nc-christian-dutton) [inline models/app-models.js:80]: Check exists not necessary here, let psql deal with it
